includes/class-wc-payments.php

```diff
@@ -778,6 +778,8 @@ public static function maybe_register_platform_checkout_hooks() {
 			add_filter( 'determine_current_user', [ __CLASS__, 'determine_current_user_for_platform_checkout' ] );
 			// Disable nonce checks for API calls. TODO This should be changed.
 			add_filter( 'woocommerce_store_api_disable_nonce_check', '__return_true' );
+			add_filter( 'woocommerce_checkout_fields', [ __CLASS__, 'platform_checkout_remove_default_email_field' ] );
+			add_action( 'woocommerce_checkout_before_customer_details', [ __CLASS__, 'platform_checkout_fields_before_billing_details' ], 20 );
```

Comment:
```suggestion
			if ( WC_Payments_Features::is_platform_checkout_enabled() ) {
				add_filter( 'woocommerce_checkout_fields', [ __CLASS__, 'platform_checkout_remove_default_email_field' ] );
				add_action( 'woocommerce_checkout_before_customer_details', [ __CLASS__, 'platform_checkout_fields_before_billing_details' ], 20 );
			}
```
We should make sure are changes are hidden behind feature flag.

Reply:
My bad. I didn't see the context.